### ISSUE TITLE
413 Request Entity Too Large

### ISSUE THREAD

**Issue by psychedelicu** [bunkerity/bunkerweb#1562]
I'm getting a 413 Request Entity Too Large error when uploading themes or plugins in WordPress through BunkerWeb reverse proxy. I've tried increasing client_max_body_size in Nginx, but BunkerWeb seems to be causing the limit. 

Does anyone know where exactly I should add **client_max_body_size 256M;** in BunkerWeb to allow larger uploads?

Thanks in advance!

**Comment by TheophileDiot** [comment]
Hi, sorry for the late response. We have a setting that might help you with your inquiry: docs.bunkerweb.io/latest/security-tuning/#max-sizes

**Comment by kmbuthia** [comment]
I am also facing this error.
I've tried adding this setting and still experiencing this exact same error. I'm using the docker (compose) autoconf setup and added the env:
`MAX_CLIENT_SIZE=20m`
But the issue persists after rebuilding the container. I am unable to upload anything larger than 10mb which seems to be the default as per the docs.

I have also tried adding modsec rules but that didn't work either. I'm also quite sure the limitation is being imposed by bunkerweb because if I point my reverse proxy to the app directly an upload of 11mb works just fine. It only fails once I direct all traffic through bunkerweb.

I'm running bunkerweb 1.5.9

**Comment by TheophileDiot** [comment]
@kmbuthia I had an issue like this recently and the problem was from WordPress and not BunkerWeb. This StackOverflow answer fixed it for me: https://stackoverflow.com/a/54013100

**Comment by kmbuthia** [comment]
Thanks for your response @TheophileDiot 

I did have this in `.ini` config place already so I had to dig more and more into the problem.
I eventually found the problem was another plugin that had enforced an upload limit via .htaccess rules. Once I changed that, everything was fine and it had nothing to do with bunkerweb. Wasted most of yesterday and a bit of today trying to figure this out!

Thanks again for the response, it is appreciated!
